### PR TITLE
RavenDB-12292: Added the ability to filter out Attachments when doing smuggler import.

### PR DESCRIPTION
**Pull request by garayx**

### CODE REVIEW

**Comment by grisha-kotler** [comment]
What about import database?

**Comment by ppekrol** [comment]
I've talked with @garayx we will add it there too. Also we will check if this (along with Counters) can be added to export.

**Comment by garayx** [comment]
> What about import database?

done

File: src/Raven.Studio/typescript/models/database/tasks/exportDatabaseModel.ts [hunk 1]
```diff
@@ -69,8 +74,14 @@ class exportDatabaseModel {
 
     private initValidation() {
         this.exportDefinitionHasIncludes = ko.pureComputed(() => {
-            return this.includeDatabaseRecord() || this.includeDocuments() || (this.includeRevisionDocuments() && this.revisionsAreConfigured()) || this.includeConflicts() ||
-                this.includeIndexes() || this.includeIdentities() || this.includeCompareExchange() || this.includeCounters();
+            if (this.includeAttachments()) {
+                this.validationMessage("Note: Attachments can only be included with documents.");
```
Comment (ml054) [inline src/Raven.Studio/typescript/models/database/tasks/exportDatabaseModel.ts:78]: Can we automatically check documents when user checks attachments + avoid having generic field: validationMessage in this model?

**Review by ml054 — CHANGES_REQUESTED** [review]
@garayx Please have a look at my comment

**Review by ppekrol — APPROVED** [review]

**Pull request merged**